common/components/ui/Identicon.jsx

```diff
@@ -17,6 +17,7 @@ export default function Identicon(props: Props) {
     : '';
   return (
     <div
+      className="Identicon"
```

Comment:
You're right, leftovers from some early styling stuff I was doing. I'll cut it out.